inc/saimacsec.h

```diff
@@ -645,6 +645,23 @@ typedef enum _sai_macsec_sc_attr_t
      */
     SAI_MACSEC_SC_ATTR_SA_LIST,
 
+    /**
+     * @brief Cipher suite for this Secure Channel.
+     *
+     * @type sai_uint64_t
+     * @flags MANDATORY_ON_CREATE | CREATE_ONLY
+     */
+    SAI_MACSEC_SC_ATTR_MACSEC_CIPHER_SUITE,
```

Comment:
In saimacsec, the ciphersuite parameters are all separately defined (e.g. SAI_MACSEC_SA_ATTR_SAK_256_BITS).  What extra information will this parameter provide the driver?

Reply:
I would propose to additionally eliminate the existing SAI_MACSEC_SA_ATTR_SAK_256_BITS and SAI_MACSEC_SC_ATTR_XPN64_ENABLE.  I am not aware of the history behind the current saimacsec definition, but there doesn't seem to be a benefit to separating the cipher suite into two attributes and putting one in the SC and the other in the SA when both are determined at the same time by the cipher suite.

Reply:
Key-length and sequence numbers are independent parameters and names clearly describe the functionality.  Both are part of cipher-suite definition for MACsec, but not for IPsec.  In future we may use the same API for other protocols.